index.rst

```diff
@@ -425,56 +425,60 @@ Simulated source spectra
 
-.. figure:: /_static/spectra/new/star_spectra_sim423_bright.png
-   :name: fig-starspectrum_bright2
 
-   Example input spectrum for a type F star with surface temperature ~6370K (solid blue).
-   The symbols are as :numref:`fig-starspectrum_bright1`.
+.. figure:: /_static/spectra/spectrum_F6909.png
+   :name: fig-starspectrum_F
 
-.. figure:: /_static/spectra/new/star_spectra_sim324_mid.png
-   :name: fig-starspectrum_mid
+   Example input spectrum for a type F star with surface temperature ~6909K (solid blue).
+   The symbols are as :numref:`fig-starspectrum_A`.
 
-   Example input spectrum for a type G  star with surface temperature ~5420K (solid blue).
-   The symbols are as :numref:`fig-starspectrum_bright1`.
+.. figure:: /_static/spectra/spectrum_G5991.png
+   :name: fig-starspectrum_G
 
-.. figure:: /_static/spectra/new/star_spectra_sim262_mid.png
-   :name: fig-starspectrum_mid2
+   Example input spectrum for a type G  star with surface temperature ~5991K (solid blue).
+   The symbols are as :numref:`fig-starspectrum_A`.
 
-   Example input spectrum for a type K  star with surface temperature ~4610K (solid blue).
-   The symbols are as :numref:`fig-starspectrum_bright1`.
+.. figure:: /_static/spectra/spectrum_K5006.png
+   :name: fig-starspectrum_K1
 
-.. figure:: /_static/spectra/new/star_spectra_sim006_faint.png
-   :name: fig-starspectrum_faint
+   Example input spectrum for a type K  star with surface temperature ~5006K (solid blue).
+   The symbols are as :numref:`fig-starspectrum_A`.
 
-   Example input spectrum for a type M star with surface temperature ~3620K (solid blue).
-   The symbols are as :numref:`fig-starspectrum_bright1`.
+.. figure:: /_static/spectra/spectrum_K3907.png
+   :name: fig-starspectrum_K2
+
+   Example input spectrum for a type K star with surface temperature ~3907K (solid blue).
+   The symbols are as :numref:`fig-starspectrum_A`.
 
 While the above spectra are representative of the typical stars in the simulation, it is helpful to look at the entire set.
 For this comparison, in :numref:`fig-colorcolor01` below I plot the simulated color between the blue and the red subfilters against the measured color, ignoring the center subfilter.
 For this example, I let the forward modeling proceed until it reached 1% convergence, which took 8 iterations.
 While there is a clear correlation between the measured and simulated spectra the slope is clearly off, with the measurements flatter than the simulations.
-If a 0.1% convergence threshold is used, however, then after 24 iterations the agreement improves (:numref:`fig-colorcolor001`).
-The example spectra in :numref:`fig-starspectrum_bright1` - :numref:`fig-starspectrum_faint` above used the 0.1% threshold.
+If a 0.1% convergence threshold is used, however, then after 50 iterations the agreement improves (:numref:`fig-colorcolor001`).
+The example spectra in :numref:`fig-starspectrum_A` - :numref:`fig-starspectrum_K2` above used the 0.1% threshold.
```

Comment:
The new figures here appear significantly less impressive than the old ones. I think that's worthy of a comment: while this version of the document is self consistent, anybody browsing through history will immediately spot that something is up. A sentence to acknowledge what changed relative to the previous version would help.